### ISSUE TITLE
discrepancies between the paper and the code

### ISSUE THREAD

**Issue by wzf2022** [liruiw/GA-DDPG#22]
Thanks for your awsome work! I have read your paper and your code, but I find that there are some discrepancies between the paper and the code:

1. In the experiments/config.py file, the flag RL_TRAIN.dagger is set to False, indicating that this code does not employ the Dagger algorithm when explore = True. I am curious whether it is necessary to change this flag to True. Additionally, I would like to know if there are any other configurations that might differ from those outlined in the paper or the config.yaml file of the provided demo model (batch_size? buffer_size?).

2. I find that the goal_reward_mask is need for calculating the grasp_aux_loss for both the critic and the actor:
`self.policy_grasp_aux_loss =  goal_pred_loss(self.aux_pred[self.goal_reward_mask, :7], self.target_grasp_batch[self.goal_reward_mask, :7] )  #agent.py L151`
 However, I've observed that when the reward is greater than 0, the goal_reward_mask is set to 1:
`self.goal_reward_mask = torch.ones_like(self.time_batch).bool() * self.reward_mask  #agent.py L246`
This implies that the code computes the grasp_aux_loss only when the reward is greater than 0 (final step of a successful grasp). Is that true?

Thanks for your great work and patience!

**Comment by liruiw** [comment]
1. Dagger would be helpful for policy training. I recommend turning it on if you want better performance.
2. Yes the goal auxiliary loss will only apply to successful goals.
